apiserver/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN go mod download
 # Build
 USER root
 WORKDIR /workspace/apiserver
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o kuberay-apiserver cmd/main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o kuberay-apiserver cmd/apiserver/main.go
```

Comment:
Can you rename the file to api-server.dockerfile? It should enable development tooling (IDEs) to recognize it as such.

Reply:
Sure

Reply:
Done